platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/collections/LongListHeap.java

```diff
@@ -51,6 +51,12 @@
 @SuppressWarnings("unused")
 public final class LongListHeap extends AbstractLongList<AtomicLongArray> {
 
+    private static final ThreadLocal<ByteBuffer> TEMP_READ_BUFFER_THREAD_LOCAL = ThreadLocal.withInitial(() -> {
```

Comment:
No need to use a thread local here. Reading from file is single threaded, it can be just a field in this class used only during initialization.